### PR TITLE
[RFR] add warning when trying to filter on a field that has not been configured

### PR DESCRIPTION
**Pull request by ThieryMichel**
fix issue #27

### CODE REVIEW

File: lib/queries/whereQuery.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export function getFieldPlaceHolder(field, value) {
 
 export function getFieldType(field, searchableFields) {
     if (!searchableFields.length) {
+        console.warn('There is no allowed fields to be searched, all filter will be ignored');
```
Comment (djhi) [inline lib/queries/whereQuery.js]: s/There is/There are/

Comment (djhi) [inline lib/queries/whereQuery.js]: s/filter/filters/

File: lib/queries/whereQuery.js [hunk 2]
```diff
@@ -33,6 +34,8 @@ export function getFieldType(field, searchableFields) {
         return 'like';
     }
 
+    console.warn(`Ignoring selector: ${field}. Allowed fields: ${searchableFields}`);
```
Comment (djhi) [inline lib/queries/whereQuery.js]: s/selector/filter/

**Comment by djhi** [comment]
The second commit comt from the previous pr right ?

**Comment by ThieryMichel** [comment]
yep

File: lib/queries/whereQuery.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ export function getFieldPlaceHolder(field, value) {
 
 export function getFieldType(field, searchableFields) {
     if (!searchableFields.length) {
+        console.warn('There are no allowed fields to be searched, all filters will be ignored');
```
Comment (Kmaschta) [inline lib/queries/whereQuery.js]: @ThieryMichel This cannot be a normal usage?
If this can be the case, there should not be a warning.

**Pull request merged**